services/venice-server/src/main/java/com/linkedin/venice/listener/ReadQuotaEnforcementHandler.java

```diff
@@ -481,7 +483,7 @@ public void handleStoreChanged(Store store) {
          * a the future version will fail in most cases, because the new topic is not in the external view at all.
          *
          */
-        this.onExternalViewChange(customizedViewRepository.getPartitionAssignments(topic));
+        this.onCustomizedViewChange(customizedViewRepository.getPartitionAssignments(topic));
```

Comment:
Is it possible to remove the function of `onExternalViewChange` or just throw exception or do nothing? Since the partition assignment triggered by EV won't contain the right state.

Reply:
Will make it do nothing, we can't remove it unless we remove it from the interface `RoutingDataChangedListener` or make it abstract.